test/dirStructure/folderTest.js

```diff
@@ -0,0 +1,106 @@
+/* eslint-env mocha */
+
+const assert = require('assert')
+const fs = require('fs')
+const fse = require('fs-extra')
+const path = require('path')
+const rimraf = require('rimraf')
+
+describe('Folder Tests', function () {
+  const appDir = path.join(__dirname, '../app/folderTest')
+  const folderPaths = require('../lib/testFolderDirPathConfig.json')
+  const paths = Object.keys(folderPaths)
+  let app
+
+  before(function () {
+    fse.ensureDirSync(path.join(appDir))
+
+    app = require('../../roosevelt')({
+      appDir: appDir,
+      generateFolderStructure: true,
+      ...folderPaths
+    })
+
+    app.initServer(function () {})
+  })
+
+  after(function (done) {
+    rimraf(appDir, (err) => {
+      if (err) {
+        throw err
+      } else {
+        done()
+      }
+    })
+  })
+
+  it('should have made a new directory for "views" based on the given parameters and be a directory', function (done) {
+    const foldertest = path.join(__dirname, `../app/folderTest/${folderPaths[paths[0]]}`)
```

Comment:
These tests can be simplified by stripping out this first check. We already have param tests that ensure our param configs are set, so these tests should just lstat for a directory based on the internal param, in this case `app.expressApp.get('params').viewsPath`